lib/telemetry.go

```diff
@@ -153,6 +155,11 @@ type TelemetryConfig struct {
 	// hcl: telemetry { dogstatsd_tags = []string }
 	DogstatsdTags []string `json:"dogstatsd_tags,omitempty" mapstructure:"dogstatsd_tags"`
 
+	// DogstatsdExitBadConnection verify connection to dogstatsd server
+	//
+	// hcl: telemetry { dogstatsd_exit_bad_connection = (true|false)
```

Comment:
This is not a blocker, but I am curious what you think about calling this `dogstatsd_verify_connection` if it's not too much trouble to change?  I saw mention of a `splunk_verify_connection` from the external ticket and thought it was simple and descriptive and wanted to get your thoughts.